puma/hlplots/aux_results.py

```diff
@@ -386,3 +390,76 @@ def plot_var_vtx_perf(
 
             plot_vtx_fakes.draw()
             plot_vtx_fakes.savefig(self.get_filename(f"{flav}_vtx_fakes_vs_{perf_var}", suffix))
+
+    def plot_track_origin_confmat(
+        self,
+        normalize: str = "all",
+        show_percentage: bool = False,
+        class_names: list | None = None,
+        title_str: str = "Track Origin Auxiliary Task\nConfusion Matrix",
```

Comment:
I think some of these optional arguments could be absorbed into kwargs and the defaults could be set further downstream. Good to have the flexibility on the normalization here though